src/vs/platform/quickinput/browser/quickInput.ts

```diff
@@ -227,4 +230,75 @@ export class QuickInputService extends Themable implements IQuickInputService {
 			})
 		};
 	}
+
+	formatPinnedItems(storageKey: string, quickPick: IQuickPick<IQuickPickItem>, storageService: IStorageService, callback?: () => Promise<void>): void {
```

Comment:
Same, I think this needs to be outside of `QuickInputService`

Reply:
waiting for Tyler's input before changing back